### PR TITLE
build faq page

### PR DESCRIPTION
**Pull request by big-javascript-fan**

### CODE REVIEW

**Comment by big-javascript-fan** [comment]
build faq page from issue #1

File: elm/src/Faq/Update.elm [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    , title = "FAQ - Flint"
+    , selectedTopic = 0
+    }
+
+
+update : Msg -> Model -> Return Msg Model
+update msg model =
+    case msg of
+        Select topicId ->
+            singleton { model | selectedTopic = topicId }
```
Comment (ailrk) [inline elm/src/Faq/Update.elm:49]: The indentation is sometimes 2 spaces sometimes 4. You can use a formatter here.

Comment (big-javascript-fan) [reply elm/src/Faq/Update.elm:49]: I just updates some with default 4 space indentation and made new commit

File: elm/src/Faq/View.elm [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+                        (Styles.paragraph
+                            ++ buttons.secondary
+                            ++ [ width <| maximum 300 fill
+                               , Font.color colors.white3
+                               , centerX
+                               , Font.center
+                               , mouseOver [ Background.color colors.gray1 ]
+                               ]
+                        )
+                        { url = "tel:+1 (844) 677-1180", label = text "+1 (844) 677-1180" }
```
Comment (ailrk) [inline elm/src/Faq/View.elm]: What are these phone numbers?

File: elm/src/Faq/View.elm [hunk 3]
```diff
@@ -0,0 +1,445 @@
...
+                                [ row
+                                    [ width fill
+                                    , Font.size 30
+                                    ]
+                                    [ text title ]
+                                , if model.selectedTopic == index then
+                                    Input.button (centerY :: Font.size 15 :: Styles.buttons.primary)
+                                      { label =
+                                          Element.image []
+                                            { src = "/static/images/github_logo.svg"
```
Comment (ailrk) [inline elm/src/Faq/View.elm]: why do we use github log here?

File: elm/src/Faq/View.elm [hunk 4]
```diff
@@ -0,0 +1,445 @@
...
+                            ]
+                    )
+            )
+        ]
+    ]
+
+
+contents : List ( String, String )
+contents =
+    [ ( "Who is this for?", "International educated Registered Nurses (RN) with a willingness to relocate to the US permanently to work as a RN." )
```
Comment (ailrk) [inline elm/src/Faq/View.elm]: Some  [questions]( https://github.com/withflint/withflint.com/issues/1) are missing

File: elm/src/Faq/View.elm [hunk 5]
```diff
@@ -0,0 +1,445 @@
...
+                    ]
+                ]
+            ]
+        ]
+    ]
+
+
+desktopLayout : Model -> List (Element Msg)
+desktopLayout model =
+    [ column
```
Comment (ailrk) [inline elm/src/Faq/View.elm]: I feel the code is too dense here, maybe abstract some rows and columns as functions?

Comment (ailrk) [reply elm/src/Faq/View.elm]: The style is not the same as the one posted on the [issue page](https://github.com/withflint/withflint.com/issues/1)

**Review by ailrk — CHANGES_REQUESTED** [review]
1. The code needs to be formatted before committing.
2. The style of the page is different from what's expected, more work is needed. 
     - the separate bar has the wrong style
     - the collapse button has the wrong style
     - font size and colour is wrong
3. The phone layout view is copied and pasted from the contact page. The view on the mobile device is broken.
4. The code in `View.elm` is too dense, maybe abstract some parts as functions.

File: elm/src/Faq/Update.elm (hunk 1 above)
Comment (ailrk) [reply elm/src/Faq/Update.elm:49]: Ok, thank you! I think more work is needed for this pr. I wrote some requested changes, please check them out :)

**Review by ailrk — CHANGES_REQUESTED** [review]
Please run `make fix` from the main directory, there are some changes that need to be made.

**Pull request closed without merging**